### PR TITLE
Plain text format mode

### PR DESCRIPTION
**Pull request by baudehlo**
I needed to restore the old format due to relying on it. This was the best way.

### CODE REVIEW

**Comment by ajinabraham** [comment]
Hi @baudehlo, I do not understand. Tabular view is still plain text and an upgrade from old hacky CLI printing. We do not want to downgrade.

However if you like to use the deprecated format or any other view,

1. You could output the results as JSON and format them as you may wish.
2. Or use njsscan Python API (https://github.com/ajinabraham/njsscan#python-api) to format the output.

**Comment by baudehlo** [comment]
It's hard to parse. So the use case is that I want the scan done, I want to
keep the full report, but also parse it to generate a summary. This older
format was much easier to parse.

If I go the JSON route I have to generate my own report afterwards. This
was less work as I'd already written the summary generator. We just had to
backtrack when it changed.

I don't mind keeping this as a separate fork. Entirely up to you if you
don't think it's valuable.

**Comment by ajinabraham** [comment]
You shouldn't be parsing non-standard ASCII output, it is bound to change and will break your parser in the long run. You should rather be parsing the output from SARIF/JSON/Python API which follows a standard. If you keep a separate fork, you will not get any rule updates or changes unless you regularly update your fork. Considering that maintenance, I would suggest you put a one time effort to parse the JSON/API out as you require.

**Comment by baudehlo** [comment]
That's fine. Can I also suggest that in future if you're going to break the
contract (the output is a form of API), you bump more than just the patch
number. That was a big change.

**Comment by ajinabraham** [comment]
Definitely, once the project matures to a major version,  it will follow semantic versioning conventions. Right now we are actively adding, improving or reworking things. Should have added a beta tag.
